pkg/kubelet/kubelet.go

```diff
@@ -1076,6 +1070,13 @@ func (kl *Kubelet) SyncPods(pods []api.BoundPod) error {
 			// syncPod() will handle this one.
 			continue
 		}
+		_, _, podAnnotations := ParsePodFullName(podFullName)
+		if source := podAnnotations[ConfigSourceAnnotationKey]; !kl.sourceReady(source) {
```

Comment:
Frankly, I am not super concerned at this point if we bounce pods.  we shouldn't be careless about it, but I would not sweat it too much.